jest.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/zipper-team/statuspage/pullRequest?prid=7096534)
```suggestion
  testEnvironment: "node",
```